plugins/woocommerce/src/Admin/API/OnboardingFreeExtensions.php

```diff
@@ -80,7 +80,7 @@ public function get_items_permissions_check( $request ) {
 	public function get_available_extensions( $request ) {
 		$extensions = RemoteFreeExtensions::get_extensions();
 		/**
-		* Remove Jetpack when woocommerce_suggest_jetpack is false.
+		* When false, Jetpack is removed from the list of suggested extensions.
```

Comment:
```suggestion
		* When false, Jetpack is removed from the list of extensions suggested in the Onboarding Profiler. This list is first retrieved from the WooCommerce.com API, then if a plugin with the 'jetpack' slug is found, it is removed.
```

Reply:
Love the improvements here! One thing to note - I think we should eventually apply the filter across all of WC Admin where Jetpack is suggested. It is also in Analytics for example ([screenshot in this issue](https://github.com/woocommerce/woocommerce/issues/37016)).

To iterate a bit on everyone's ideas, my suggestion is:

```
* Allows removing Jetpack suggestions from WooCommerce Admin when false.
*
* In this instance it is removed from the list of extensions suggested in the Onboarding Profiler. This list is first retrieved from the WooCommerce.com API, then if a plugin with the 'jetpack' slug is found, it is removed
```

Reply:
Thank you! 😄  @adrianduffell @rjchow Updated in e84f289b1dc11fadf5d304a5bf5069e49246af16.